jobs/furnishings/src/furnishings/config.py

```diff
@@ -84,6 +84,8 @@ class _Config:  # pylint: disable=too-few-public-methods
     TESTING = False
     DEBUG = False
 
+    SECOND_NOTICE_DELAY = os.getenv('SECOND_NOTICE_DELAY', None)
```

Comment:
can we do the int casting here like we do in dissolutions job?  Think it's safe to just default to 5 as well if nothing returns for whatever reason.

https://github.com/bcgov/lear/blob/2685686f797871a80267f110401f3cbfde59e911/jobs/involuntary-dissolutions/config.py#L84